script/process_qrc.py

```diff
@@ -23,31 +23,76 @@
 
 """
 
+# Standard library imports
 from __future__ import absolute_import, print_function
-
+from subprocess import call
 import argparse
 import glob
 import os
 import sys
-from subprocess import call
+
+# Third party imports
+from watchdog.events import FileSystemEventHandler
+from watchdog.observers import Observer
+
+# Constants
+HERE = os.path.abspath(os.path.dirname(__file__))
+REPO_ROOT = os.path.dirname(HERE)
+
+
+class QSSFileHandler(FileSystemEventHandler):
```

Comment:
This watcher (with a small tweak) could also be applied to other scripts.

Reply:
Nice! Tks